### PR TITLE
Command palette

### PR DESCRIPTION
**Pull request by yuval-hazaz**
- [x] new style
- [x] prioritzie current app results

### CODE REVIEW

File: packages/amplication-client/src/CommandPalette/CommandPalette.tsx [hunk 1]
```diff
@@ -33,13 +43,43 @@ export class NavigationCommand implements Command {
...
+    public readonly showAppData: boolean,
+    public readonly appName?: string,
+    public readonly appColor?: string
   ) {}
   command() {
     this.history.push(this.link);
   }
 }
 
+function RenderCommand(suggestion: Command) {
```
Comment (iddan) [inline packages/amplication-client/src/CommandPalette/CommandPalette.tsx:58]: Name of the component should not include "render"

File: packages/amplication-client/src/CommandPalette/CommandPalette.tsx [hunk 2]
```diff
@@ -33,13 +43,43 @@ export class NavigationCommand implements Command {
...
+    <>
+      {showAppData && (
+        <>
+          <ApplicationIcon name={appName || ""} color={appColor} />
+          <span className="command-palette__app-name">{appName}</span>
+        </>
+      )}
+      <Icon icon={type} />
+      {highlight && highlight[0] ? (
+        <span dangerouslySetInnerHTML={{ __html: highlight[0] }} />
```
Comment (iddan) [inline packages/amplication-client/src/CommandPalette/CommandPalette.tsx:71]: What if you have multiple highlights?

File: packages/amplication-client/src/CommandPalette/CommandPalette.tsx [hunk 3]
```diff
@@ -109,6 +156,21 @@ const CommandPalette = ({ trigger }: Props) => {
...
+        allowTypo: true,
+        scoreFn: (item) => {
+          const command: NavigationCommand = item.obj;
+          const scoreFactor = command.isCurrentApp ? 1000 : 0;
+
+          return Math.max(
+            item[0] ? item[0].score + scoreFactor : -1000,
+            item[1] ? item[1].score + scoreFactor : -1000
+          );
+        },
```
Comment (iddan) [inline packages/amplication-client/src/CommandPalette/CommandPalette.tsx:171]: Extract to a function at the top level and add a unit test

File: packages/amplication-client/src/CommandPalette/CommandPalette.tsx (hunk 2 above)
Comment (yuval-hazaz) [reply packages/amplication-client/src/CommandPalette/CommandPalette.tsx:71]: The first highlight is for the "name" property, the second one is for the appName. I thought there is no benefit from highlighting the app name. 
I will have a session with @Leeyaacov tomorrow and if needed, I will add it.

**Pull request merged**